python-libraries/data-platform-catalogue/data_platform_catalogue/client/datahub/datahub_client.py

```diff
@@ -664,6 +665,8 @@ def get_table_details(self, urn) -> TableMetadata:
             domain = parse_domain(response)
             owner, owner_email = parse_owner(response)
             tags = parse_tags(response)
+            name = properties["name"]
```

Comment:
This is throwing an `KeyError`when on run for dbt dataset urn. It's something that has slipped through,`name` is not set within the `properties` key of the response for dbt ingestions. They have a `name` key/value at the top level of the response, however this is equivalent to a fully qualified name e.g. `'awsdatacatalog.dbt_database.dbt_table'

Reply:
Ah, annoying. I've added a fallback to top level `name` to match the search response.